package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "pied_nike",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "prepare": "husky",
+    "lint": "eslint --ext .ts,.tsx src/",
+    "lint:fix": "eslint --fix --ext .ts,.tsx src/",
+    "format:check": "prettier --check src --cache",
+    "format:write": "prettier --write src --cache",
+    "postinstall": "husky install",
```

Comment:
Do we need them?

Reply:
I turn on "Format on save", so it doesn't make sense to me. But maybe my teammate don't, so he/she can use this script to format code. --cache to ensure Prettier formats files only if they have changed.

Reply:
act, I don't think it needs. you just need to care about formated code before commit